source/upgrade/important-upgrade-notes.rst

```diff
@@ -5,14 +5,20 @@ Important Upgrade Notes
   :start-after: :nosearch:
 
 .. important::
-   - Support for Mattermost Server v6.3 :doc:`Extended Support Release </upgrade/extended-support-release>` has come to the end of its life cycle in October 15, 2022. Upgrading to Mattermost Server v7.1 :doc:`Extended Support Release </upgrade/extended-support-release>` or later is recommended.
+   - Support for Mattermost Server v7.1 :doc:`Extended Support Release </upgrade/extended-support-release>` is coming to the end of its life cycle in May 15, 2023. Upgrading to Mattermost Server v7.8 :doc:`Extended Support Release </upgrade/extended-support-release>` or later is recommended.
    - MySQL 8.0.22 contains an `issue with JSON column types <https://bugs.mysql.com/bug.php?id=101284>`__ changing string values to integers which is preventing Mattermost from working properly. Users are advised to avoid this database version.
    - Upgrading the Microsoft Teams Calling plugin to v2.0.0 requires users to reconnect their accounts.
    - When upgrading to 7.x from a 5.x release please make sure to upgrade to 5.37.10 first for the upgrade to complete successfully.
 
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v7.9                                               | Added a new API endpoint ``GET api/v4/posts/[POST_ID]/edit_history``. Users can run "CREATE INDEX CONCURRENTLY" before doing the upgrade to allow PostgreSQL to  |
+|                                                    | create the index without locking the table.                                                                                                                      |
```

Comment:
Same comment as before. We can use:
I just updated that PR description with times for the index creation. Can we use this?

>Added a new API endpoint `GET api/v4/posts/[POST_ID]/edit_history`.
>Added a new index on `Posts(OriginalId)`. For a database with 11.8 million posts, on a machine with a i7-11800H CPU (8 cores, 16 threads), 32GiB of RAM and SSD, the index creation takes:
>- MySQL: 98.51s
>- PostgreSQL: 2.6s
>
> To avoid any table locking, servers using Postgres are recommended to create the index online before performing the upgrade using the `CONCURRENTLY` option, as in 
>```CREATE INDEX CONCURRENTLY idx_posts_original_id ON Posts(OriginalId);```